From cae68083fda5d4ca832ff3cc8a533454df2efe23 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 12 Oct 2021 20:35:53 -0700
Subject: [PATCH] libknet/tests: Correct include path for poll.h

Fixes
/usr/include/sys/poll.h:1:2: error: redirec
ting incorrect #include <sys/poll.h> to <poll.h> [-Werror,-W#warnings]
| #warning redirecting incorrect #include <sys/poll.h> to <poll.h>

Upstream-Status: Submitted [https://github.com/kronosnet/kronosnet/pull/363]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 libknet/tests/test-common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libknet/tests/test-common.c b/libknet/tests/test-common.c
index 86b76b0..8f8b6ca 100644
--- a/libknet/tests/test-common.c
+++ b/libknet/tests/test-common.c
@@ -20,7 +20,7 @@
 #include <pthread.h>
 #include <dirent.h>
 #include <sys/select.h>
-#include <sys/poll.h>
+#include <poll.h>
 
 #include "libknet.h"
 #include "test-common.h"
